fusesoc/provider/git.py

```diff
@@ -1,11 +1,30 @@
 import logging
 import shutil
+import os.path
 from fusesoc.provider.provider import Provider
 from fusesoc.utils import Launcher
 
 logger = logging.getLogger(__name__)
 
 class Git(Provider):
+    @staticmethod
+    def init_library(library):
+        if not os.path.isdir(library['location']):
```

Comment:
Perhaps we should notify the user somehow that we didn't clone the repo. Raise RuntimeError perhaps?

Reply:
Changed this to re-raise any Launcher errors as RuntimeErrors. I figure we can specify that init_library raises RuntimeErrors regardless of how the provider is implemented.